document-service/documents-ui/src/components/documentsTable/index.vue

```diff
@@ -58,14 +181,28 @@ const openDocumentRecord = (searchResult: DocumentInfoIF) => {
           <span
             v-for="(file, i) in row.consumerFilenames"
             :key="`file-${i}`"
-          >
+            class="block my-1"
+          >  
              <ULink
-               inactive-class="text-primary underline"
+               inactive-class="text-primary"
+               class="flex align-center"
                @click="downloadFileFromUrl(row.documentUrls[i], file)"
              >
+              <UIcon name="i-mdi-file-download" class="w-5 h-5" />
               {{ file }}
             </ULink>
           </span>
+          <span
+            v-if="row.consumerFilenames.length > 1"
+          >
```

Comment:
Very minor, but general rule of thumb:
if only 1 prop on an element, can single line it
if multiple: multi lines

no biggie though, just for reference